### PR TITLE
SWC-5956

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/main/java/org/sagebionetworks/web/client/widget/table/explore/TableEntityPlotsWidget.java [hunk 1]
```diff
@@ -0,0 +1,671 @@
+package org.sagebionetworks.web.client.widget.table.explore;
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/table/explore/TableEntityPlotsWidget.java:1]: * Initially copied from TableEntityWidget with the following modifications.
 * Uses SRC QueryWrapperPlotNav
 * Removed Query Input widget.  Removed concept of simple and advanced search.
 * This is responsible for editing table rows.
 * Copied isQueryResultEditable() from TableQueryResultWidget.

File: src/main/java/org/sagebionetworks/web/client/widget/table/explore/TableEntityPlotsWidget.java [hunk 2]
```diff
@@ -0,0 +1,671 @@
...
+		this.actionMenu.setActionVisible(Action.EDIT_TABLE_DATA, false);
+		this.actionMenu.setActionVisible(Action.DOWNLOAD_TABLE_QUERY_RESULTS, false);
+	}
+
+	/**
+	 * The results are editable if all of the select columns have ID
+	 * 
+	 * @return
+	 */
+	public boolean isQueryResultEditable(QueryResultBundle bundle) {
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/table/explore/TableEntityPlotsWidget.java]: Can this be static, and moved to the row editor?

Comment (jay-hodgson) [reply src/main/java/org/sagebionetworks/web/client/widget/table/explore/TableEntityPlotsWidget.java]: (with tableType param)

File: src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidgetViewImpl.java [hunk 3]
```diff
@@ -56,6 +58,8 @@
 	@UiField
 	Div modalContainer;
 	@UiField
+	Div plotNavContainer;
```
Comment (nickgros) [inline src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidgetViewImpl.java:61]:
```suggestion
	QueryWrapperPlotNav plotNavContainer;

```

Comment (nickgros) [reply src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidgetViewImpl.java:61]: Would have to move constructor logic to other methods...not really worth it

File: src/main/java/org/sagebionetworks/web/client/widget/table/v2/results/TableQueryResultWikiWidget.java [hunk 4]
```diff
@@ -132,8 +149,13 @@ public void hideEditActions() {
 		this.actionMenu.setActionVisible(Action.EDIT_TABLE_DATA, false);
 		this.actionMenu.setActionVisible(Action.SHOW_TABLE_SCHEMA, false);
 		this.actionMenu.setActionVisible(Action.SHOW_VERSION_HISTORY, false);
+		GWT.debugger();
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/table/v2/results/TableQueryResultWikiWidget.java]: Remove

File: src/test/java/org/sagebionetworks/web/unitclient/widget/table/explore/TableEntityPlotsWidgetTest.java [hunk 5]
```diff
@@ -0,0 +1,698 @@
...
+		((Dataset) entityBundle.getEntity()).setIsLatestVersion(false);
+		versionNumber = 1L;
+
+		widget.configure(entityBundle, versionNumber, canEdit, mockQueryChangeHandler, mockActionMenu);
+
+		verify(mockActionMenu).setTableDownloadOptionsEnabled(true);
+	}
+
+	@Test
+	public void testTotalVisibleResultsNotConfiguredForTableEntity() {
```
Comment (jay-hodgson) [inline src/test/java/org/sagebionetworks/web/unitclient/widget/table/explore/TableEntityPlotsWidgetTest.java]: nonsensical, remove

**Review by nickgros — APPROVED** [review]

**Pull request merged**